loadtest/results/cross-vm-affinity-probe-2026-08-28/README.md

```
@@ -0,0 +1,117 @@
+| rig | 이 디렉토리의 [`affinity_probe.py`](./affinity_probe.py), VM B의 컨테이너 안에서 `docker exec`로 실행 |
+| 소요 | 부트스트랩(docker build 포함) ~10분 + 실측 ~5분 ≈ **15분** |
+
+---
+
+## 2. 판정선별 결과
+
+### ㄱ. 정확성 — 세션이 항상 배정된 VM으로만 가는가
+
+킬 전(`frames pre`), 각 VM에 배정된 세션 10개씩 프레임을 보냈다:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)